src/pages/settings/InitialSettingsPage.js

```diff
@@ -286,7 +286,7 @@ function InitialSettingsPage(props) {
     const accountID = lodashGet(currentUserDetails, 'accountID', '');
 
     const headerContent = (
-        <View style={[styles.avatarSectionWrapperSettings, styles.justifyContentCenter]}>
+        <View style={[styles.avatarSectionWrapperSettings, styles.justifyContentCenter, styles.pl6, styles.pr3]}>
             {_.isEmpty(props.currentUserPersonalDetails) || _.isUndefined(props.currentUserPersonalDetails.displayName) ? (
```

Comment:
Why left and right paddings are different? Is it consulted with a designer somewhere?

<img width="434" alt="Screenshot 2024-01-25 at 13 20 53" src="https://github.com/software-mansion-labs/expensify-app-fork/assets/104823336/c3a5b63d-e4b2-4995-8bd9-80c35365c619">

Reply:
Ok, I've consulted it with Shawn (designer), and the padding should be 20px horizontal.
```suggestion
        <View style={[styles.avatarSectionWrapperSettings, styles.justifyContentCenter, styles.ph5]}>
            {_.isEmpty(props.currentUserPersonalDetails) || _.isUndefined(props.currentUserPersonalDetails.displayName) ? (
```